parm/atm/obs/config/amsua_n19.yaml

```diff
@@ -0,0 +1,323 @@
+obs space:
+  name: amsua_n19
+  obsdatain:
+    obsfile: $(OBS_DIR)/$(OBS_PREFIX).amsua_n19.$(OBS_DATE).nc4
+  obsdataout:
+    obsfile: $(DIAG_DIR)/diag_amsua_n19_$(OBS_DATE).nc4
+  simulated variables: [brightness_temperature]
+  channels: &amsua_n19_channels 1-15
+obs operator:
+  name: CRTM
+  Absorbers: [H2O,O3,CO2]
```

Comment:
Does CO2 work in the variational analysis?   It didn't before.

Reply:
Good point, I forgot about this, I bet it still doesn't work properly without the 'linear obs operator' part added.